src/burnchains/burnchain.rs

```diff
@@ -365,6 +365,35 @@ impl Burnchain {
         })
     }
 
+    /// the expected sunset burn is:
+    ///   total_commit * (progress through sunset phase) / (sunset phase duration)
+    pub fn expected_sunset_burn(&self, burn_height: u64, total_commit: u64) -> u64 {
```

Comment:
`burn_height` is a stacks block height, not a burnchain block height, correct?

Reply:
It's a burnchain block height -- PoX sunset timelines and reward cycles (and locks/unlocks) are governed by the burnchain block height, not the stacks height.